translations/tgl.po

```diff
@@ -1269,11 +1269,11 @@ msgstr "Manatiling buhay sa loob ng 32 segundo"
 
 #: ../client/application/game/levels/game_levels.cpp:120
 msgid "\"This level is a joke right?\" - a tester"
-msgstr "\"Totoo ba 'to?\" - kawawang tester :("
+msgstr "\"ofivefive, how to be u po?\" - noobie213"
```

Comment:
What happened to the tester? 
I don't know about this. 😄

Reply:
In this translation, it's more like I change the whole context of this part of string, but still connected to Fury. Here's the context:
![image](https://user-images.githubusercontent.com/78365976/203100238-4d37c0d4-9daa-4712-a0be-5ea92b40e229.png)

I don't know tho if I'm allowed to do this. I'll just revert it back to the translation before if this isn't allowed.

Reply:
I see a few issues with doing this.
- Players who are Filipino but aren't in the Philippines currently won't understand that
- There is a need to update the translation if someone gets a higher score

And I think this just has a whole different meaning that is not even related to the English string

Reply:
Oh, great pointers. Yup, probably better to stay to my translation before lol

Reply:
Alright, I'm marking these are resolved. Make the changes any time and I'll approve them.